thunder/tests/opinfos.py

```diff
@@ -2496,6 +2496,31 @@ def addcmul_addcdiv_sample_generator(op, device, dtype, requires_grad, **kwargs)
 opinfos.append(addcdiv_opinfo)
 
 
+def lerp_sample_generator(op, device, dtype, requires_grad, **kwargs):
+    S = 4
+    cases = (
+        ((S,), (S,), (S,)),
+        ((S, S), (S, S), (S, S)),
+        ((S, 1), (1, S), (S, 1)),
```

Comment:
What happens if the length of a dimension is zero?

Reply:
Since `lerp` is an elementwise operation, nothing particular happens when the inputs are 0D. I thought we did not need to test 0D inputs specifically on `lerp`.

Reply:
But I added 0D inputs anyway, having various test cases is nice.